pkg/convert/xliff2_test.go

```diff
@@ -109,6 +112,33 @@ func Test_FromXliff2(t *testing.T) {
 			data:     randXliff2(nonOriginalTranslation),
 			expected: nonOriginalTranslation,
 		},
+		{
+			name:                "Message with special chars",
```

Comment:
`containsSpecialChar: true` makes the test less understandable, is there any way to remove it?

Reply:
about the name of the test, a Message with special chars also should work fine, cuz it should escape also if there is only a single ("{"or "}" or "|" or "\\") special char inside. (escapeSpecialChars func in convert.go)
can you please explain how containsSpecialChar flag makes it less understandable?  :)

Reply:
well if it escapes those as well then I rest my case :D